Sources/SwifterSwift/Foundation/MeasurementExtensions.swift

```diff
@@ -0,0 +1,52 @@
+// MeasurementExtensions.swift - Copyright 2021 SwifterSwift
+
+#if canImport(Foundation)
+import Foundation
+
+// MARK: - Methods
+@available(OSX 10.12, tvOS 10.0, watchOS 3.0, *)
+public extension Measurement where UnitType == UnitAngle {
+    /// SwifterSwift:  Create a Measurement for an angle with a specified value in degrees.
```

Comment:
Nit: Normally on documentation class names are `Measurement` 
```suggestion
    /// SwifterSwift:  Create a `Measurement` for an angle with a specified value in degrees.
```

Reply:
Thanks!